docker/bin/zookeeperStart.sh

```diff
@@ -51,6 +51,22 @@ if [ -f $DYNCONFIG ]; then
   ONDISK_DYN_CONFIG=true
 fi
 
+# Check if envoy is up and running
```

Comment:
Can we remove this, since we are  waiting for  40s in nslookup?

Reply:
For debugging reasons I think it is better to have it separately, and it does not adds more wait time to the startup time. Moreover, it is only affects clusters where Istio is enabled.